.gitignore

```diff
@@ -14,3 +14,4 @@ Gemfile.lock
 .ruby-gemset
 *.retry
 *.pyc
+user_playbooks
```

Comment:
Does gitkeep prevent you from committing anything you add to it? I want to help prevent devs accidentally committing their custom scripts.